source/docs/articles/sites/migrate/wordpress-site-networks.md

```diff
@@ -0,0 +1,120 @@
+---
+title: Migrate to Pantheon: WordPress Site Networks
+description: Learn how to import a WordPress Site Network into Pantheon.
+keywords: import, importing site, pantheon, new site, large site, distro, upstream, git history
+---

```

Comment:
@bmackinney do we need a new doc for this? I'm not sure there are any differences between the instructions here and the ones found on the manual site import doc.